.pre-commit-config.yaml

```diff
@@ -39,6 +39,14 @@ repos:
     files: ^.*\.pkr\.hcl$
     pass_filenames: true
     require_serial: true
+  - id: duplicate-diff
+    name: duplicate-diff
+    entry: tools/duplicate-diff.sh
+    language: script
+    files: ^.*\.sh$
```

Comment:
you might use `types: [shell]` here.

https://github.com/pre-commit/identify/blob/3a9071650b0ee34151f6d7293122f7cc4fe63562/identify/extensions.py#L191

Reply:
updated.